packages/algob/src/internal/deployer.ts

```diff
@@ -331,6 +346,13 @@ export class DeployerDeployMode extends DeployerBasicMode implements Deployer {
     this.cpData.registerSSC(this.networkName, sscName, sscInfo);
   }
 
+  /**
+   * Register UpdatedSSC Info in checkpoints
+   */
+  registerUpdatedSSCInfo (sscName: string, sscInfo: SSCInfo): void {
+    this.cpData.registerUpdatedSSC(this.networkName, sscName, sscInfo);
+  }
```

Comment:
Is it possible to use `registerSSCInfo` while updating SSC as well? I think we create a checkpoint even when we update an ssc, so maybe we should reuse registerSSCInfo?

Reply:
i thought so, but having two different functions seems clear and easily understandable to developers, I can change to 
combining into functions if you want. also reuse part is only setting values in map.

Reply:
yes, then i think we could make this method private (`_registerUpdatedSSCInfo`), and move it's usage to *registerSSCInfo*. So during deploy/update ssc, only *registerSSCInfo* is used (which internally uses *this._registerUpdatedSSCInfo*, during app update). So it would be understandable as well